exampleCase/gstTestBackground/runTest.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+if [[ "$SSATAPILOADED" != "1" ]]
+then
+	SILENT=0
+	INDEPENDENT=1
+	search="ssat-api.sh"
+	source $search
+
+	retcode=$?
+	count=0
+	while (( ${retcode} != 0 ))
+	do
+		count=$((count+1))
+		if (( ${count} > 5 ))
+		then
+			echo "Cannot find ssat-api.sh"
+			exit 1
+		fi
+
+		search="../${search}"
+		source $search
+		retcode=$?
+	done
+	printf "${Blue}Independent Mode${NC}\n"
+fi
+
+testInit $1
+
+command -v gst-launch-1.0 || report
+command -v gst-inspect-1.0 || report
+
+filename=$(mktemp)
+gstTestBackground "videotestsrc num-buffers=5 is-live=true  ! video/x-raw,width=640,height=480,framerate=1/1 ! filesink location=${filename}" GBKG1 0 0 5
```

Comment:
line 34 and 42: 320x240 to reduce mem size?
(videotestsrc finds the nearest resolution 320-240)

Reply:
Updated to 64x48. The size doesn't matter for this test case. :)